teccloud-api/package.json

```
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "nodemon --watch src --exec yarn start",
-    "start": "ts-node src/main.ts",
+    "dev": "ENVIRONMENT=dev nodemon --watch src --exec 'ts-node src/main.ts'",
+    "start": "ENVIRONMENT=prod ts-node src/main.ts",
```

Ando en windows, entonces un dev pa nosotros. Jaja
```suggestion
    "dev": "ENVIRONMENT=dev nodemon --watch src --exec 'ts-node src/main.ts'",
    "dev-windows": "set ENVIRONMENT=dev && nodemon --watch src --exec ts-node src/main.ts",
    "start": "ENVIRONMENT=prod ts-node src/main.ts",
```

---

No ma, no sabía que no jalaba en Windows.
Ya le puse una que se llama `dev-win`